### ISSUE TITLE
parameters selection

### ISSUE THREAD

**Issue by molinfzl** [mcfrith/last-genome-alignments#21]
Hi,I have been doing the alignment between human and mole recently, but I am confused by the many parameter choices. I have observed that you provided the  alignment parameters between human and mouse in 2017 based on a lower version of the software, whether this is still applicable.

I am currently using latest lastal 1543, my command:

lastdb -P8 -uMAM8 -c myDB genome1.fa

last-train -P8 --revsym -D1e9 --sample-number=5000 myDB genome2.fa > my.train

lastal -P8 -D1e9 -m100 --split-f=MAF+ -p my.train myDB genome2.fa > many-to-one.maf

last-split -r many-to-one.maf > one-to-one.maf

Is that too strict and too slow, because they are mammals, do you have any suggestions or any appropriate parameters for that?

best regards

**Comment by mcfrith** [comment]
Yes, the many parameters are a big problem, sorry about that. Please ignore the old 2017 parameters. (I should probably "hide" them in a separate file.) The latest recommendations are here: https://gitlab.com/mcfrith/last/-/blob/main/doc/last-cookbook.rst

In short, I suggest running lastdb like this:
`lastdb -P8 -c mydb genome1.fa`
Then follow the "Aligning human & chimp genomes" recipe.
